api/RS.API/RS.Data/Interfaces/IApprovalRepository.cs

```diff
@@ -1,10 +1,15 @@
 ﻿using System.Collections.Generic;
 using RS.Entity.Models;
+using RS.ViewModel.User;
 
 namespace RS.Data.Interfaces
 {
     public interface IApprovalRepository : IRepository<Approvals>
     {
         List<ApprovalEvents> GetApprovalEvents(int approvalId);
+        List<ApprovalEvents> GetAllApprovalEvents();
```

Comment:
filter result by approval id or remove this function

Reply:
Done